### ISSUE TITLE
Control C Data from 8IOL AOI

### ISSUE THREAD

**Issue by nickcluteturck** [nickcluteturck/IOLinkAOIs#1]

**Comment by nickcluteturck** [comment]
Need to make sure we're inhibiting/uninhibiting the block after a change if possible....not sure how well an SSV command can get wrapped into an AOI?

**Comment by nickcluteturck** [comment]
Accidentally closed this issue as I was figuring out the Github UI on how to comment correctly...apparently you just click the green button, I thought the dropdown next to it was a modifier to the green button....

**Comment by jose-sanchez-tcan** [comment]
Ok, so this one we decided to have Config tags in and config tags out, do we still issue a inhibiting/uninhibiting after a change on the config tags?

**Comment by nickcluteturck** [comment]
No, I'm good with out happening naturally unless they read the manual.
________________________________
